src/http-gateways/trustless-gateway.md

```diff
@@ -155,16 +155,16 @@ The following additional values are supported:
 A Gateway MUST augment the returned `Etag` based on the passed `entity-bytes`.
 
 A Gateway SHOULD return an HTTP 400 Bad Request error when the requested range
-cannot be parsed as valid offset positions.
+is outside of valid offset positions in full, and the gateway knows that upfront.
```

Comment:
soft suggestion, my brain tripped up on "in full" when reading this on the first pass.

```suggestion
is entirely outside of the entity's byte range and the server is able to determine this
upfront.
```